Misc/NEWS.d/next/Core and Builtins/2024-08-13-21-40-20.gh-issue-122982.JSLp1k.rst

```diff
@@ -0,0 +1,3 @@
+Bitwise inversion (``~``) is prohibited for operand of type :func:`bool`.
```

Comment:
```suggestion
Bitwise inversion (``~``) is prohibited for operands of type :func:`bool`.
```

Sorry I missed this one...

Reply:
I'm not a fluent English speaker, but, is this true? Since unary operators have exactly one operand, I think that this is correct.

Reply:
Then I would have used "an operand".

Let's ask a native one: @AA-Turner please help us!

Reply:
I would also have used `class` because of its semantics in this case.

Reply:
I'd phrase it as either "Bitwise inversion (``~``) is prohibited for Boolean operands." or "Bitwise inversion (``~``) is prohibited for :data:`True` and :data:`False`.

By the way, you can tag @python/proofreaders if you want help with wording suggestions, such as in this case!

A

Reply:
Oh, yes, I forgot about this group!